examples/ai_functions/chuck_norris.py

```diff
@@ -0,0 +1,26 @@
+from marvin import ai_fn
+import os
+
+@ai_fn
+def chuck_norris_joke_of_the_day(howmany: int = 3, modifier : str = "") -> list[str]:
+    if modifier is not None:
```

Comment:
This conditional probably isn't working exactly as intended (which is an open point of confusion with AI Fns) since the LLM is actually seeing all the source code and not the output of this but it's probably enough that it works anyway!